docs/Protocol messaging.md

```diff
@@ -1,8 +1,7 @@
 # Protocol messaging
 
-All protocol messages MUST have have the following 2 properties:
+All protocol messages MUST have have the following property:
```

Comment:
Given this is now a list of one, could this be collapsed into a single sentence?
```suggestion
All protocol messages MUST have the messageType property to indicate the type of the message (e.g. Command).
```